jerry-core/jcontext/jcontext.c

```diff
@@ -19,6 +19,50 @@
  * @{
  */
 
+#ifdef JERRY_ENABLE_EXTERNAL_CONTEXT
+
+/**
+ * Get the current context from the current instance.
+ *
+ * @return the pointer to the context
+ */
+jerry_context_t *
+jerry_context_get_current_context (void)
```

Comment:
The .text section size: "current": 775134 ==> "inline": 775230 (+100) 

And I also measure the function size by `nm build/bin/jerry -S | grep jerry_get_user_context `
```
void *
jerry_get_user_context (void)
{
  return JERRY_CONTEXT (user_context_p);
} /* jerry_get_user_context */
```
because it's simple and only contain the JERRY_CONTEXT 

"current": 18byte
"inline": 22byte

So in the tradeoff between perf and binary size, `inline` is  prone to perf. 
What should we choose?